torchci/scripts/backfillJobs.mjs

```diff
@@ -134,8 +134,8 @@ WHERE
     AND w.repository.name = 'pytorch'
     AND j.backfill IS NULL
 ORDER BY
-    j._event_time DESC
-LIMIT 10000
+    j._event_time ASC
```

Comment:
This change the backfill order from newest first to oldest first.  I couldn't think of any side effects, so that's probably ok